### DISCUSSION TITLE
Any new or old working Securly Disablers?

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#889] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So basically my school has blocked everything to all Chrome :// links and I recently found the #translate.google.com but it doesn't work now,I tried the insecurly for a bit until they just somehow blocked that too (Sbiox or whatever it called doesn't work) so please help (Note: I can't do proxies or LTMEET, etc.. either) SO please help me by providing links which I can research. Thanks, CGRmags

**Comment by PenguinLord561** [comment]
@CGRmags I have the same problem, instead I can use chrome:// links and the flood version of ltmeat still works

**Reply by CGRmags** [reply]
@PenguinLord561 could you tell me how flood LTMEET works maybe I can try and what Chrome:// does it need

**Reply by AshtonDavies** [reply]
> @PenguinLord561 could you tell me how flood LTMEET works maybe I can try and what Chrome:// does it need 
> 

https://github.com/3kh0/ext-remover/discussions/671

**Reply by CGRmags** [reply]
ok but do you have any unblockers that can run Xbox cloud gaming? I cant the corky exploit either

**Reply by AshtonDavies** [reply]
> ok but do you have any unblockers that can run Xbox cloud gaming? I cant the corky exploit either 

LTMEAT Flood works for that if done correctly. Either figure that out, which is the better option, or use my Bypass DNS (94.140.14.49) and [link IP](https://linkip.adguard-dns.com/linkip/d4cfbd43/MrXXtcXx4U4QvPOyloGK4Y3gdER6FjJrYAiHdQoehux). Those both are not designed for extensions other than Lightspeed, but they both work on the latest version.

**Reply by CGRmags** [reply]
@AshtonDavies Can you send me a web video of you'd doing LTMEET flood and how long does it last

**Reply by AshtonDavies** [reply]
> @AshtonDavies Can you send me a web video of you'd doing LTMEET flood and how long does it last 

Potentially indefinitely. I have already posted a video in my discussion. Note that was back when 600 tabs was ideal. Please use 1000 at the minimum for a more permanent freeze.

https://github.com/3kh0/ext-remover/discussions/671#discussioncomment-6770771

**Reply by CGRmags** [reply]
@AshtonDavies when you do the folder i see the extenuion page what are the other ones and can you get me thank same link but for securely?

**Reply by AshtonDavies** [reply, marked as answer]
> @AshtonDavies when you do the folder i see the extenuion page what are the other ones and can you get me thank same link but for securely?

Grab your Securly extension page URL from [here](https://chrose.netlify.app/detail/LTMEAT%20Flood). Also add the settings page to the beginning of the folder if you haven't already done so.

**Reply by Aka-but-nice** [reply]
I still cannot get through how cringe this is as an exploit, it's practically a very watered down LTMEAT, a pathetic one if you will. Give it up :pray:.

**Reply by DankFireR** [reply]
@Aka-but-nice but it works, so what's the problem? wdym "give it up"?

**Reply by AshtonDavies** [reply]
> I still cannot get through how cringe this is as an exploit, it's practically a very watered down LTMEAT, a pathetic one if you will. Give it up :pray:.

Literally LTMEAT triggered naturally by flooding tabs, and it works, so what's the problem? We don't need you coming into these discussions just to mock kids for everything they are doing.

**Comment by CGRmags** [comment]
@AshtonDavies WOW, This worked thanks for telling me :)
